### ISSUE TITLE
No keyboard

### ISSUE THREAD

**Issue by notbadplayer** [yuliskov/SmartTube#1370]
Hello, I have issue, there is no keyboard in search panel. I have Sony Bravia TV with android tv. Voice search works good, but I there's no keyboard visible.

**Comment by vvijayk1999** [comment]
Yes, I'm facing the same issue on my Sony Bravia 4K TV. I'm using the stable [v16.04_r](https://github.com/yuliskov/SmartTubeNext/releases/download/16.04/STubeNext_ststable_v16.04_r.apk) version of SmartTubeNext.
